### PR TITLE
Remove statement about ES module browser support

### PR DESCRIPTION
**Pull request by taymonbeal**
[All modern browsers](https://caniuse.com/#feat=es6-module) support them now.

### CODE REVIEW

**Comment by montogeek** [comment]
Thanks for your contribution. I think this note is still necessary, we have to acknowledge the existence of other browsers and specially those that doesn't support features.
Something like this would be better:

> Although they are not supported by all browsers, some modern browsers do, webpack does support them out of the box.

File: src/content/guides/getting-started.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ W> If you are getting a syntax error in the middle of minified JavaScript when o
 
 ## Modules
 
-The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/docs/en/learn/). Although they are not supported in most browsers yet, webpack does support them out of the box.
+The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/docs/en/learn/). webpack supports them out of the box.
```
Comment (EugeneHlushko) [inline src/content/guides/getting-started.md]:
```suggestion
The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/docs/en/learn/). Although they are not supported in some browsers yet, webpack supports them out of the box.
```

**Comment by montogeek** [comment]
@taymonbeal Friendly ping

**Comment by chenxsan** [comment]
Closed as they're already improved as per https://github.com/webpack/webpack.js.org/blob/master/src/content/guides/getting-started.md#modules

**Pull request closed without merging**